eternalcore-plugin/run/plugins/EternalCore/lang/en_messages.yml

```
@@ -0,0 +1,669 @@
+container:
+  # These messages are sent when the player opens a container
+  # {PLAYER} - Player who opened the container
+  genericContainerOpened: []
+  genericContainerOpenedBy: "<green>► <white>The specified container has been opened by <green>{PLAYER}<white>!"
+  genericContainerOpenedFor: "<green>► <white>The specified container has been opened for <green>{PLAYER}<white>!"
+
+# Information sent, when the language is changed to English
+language:
+  languageChanged: "<green>► <white>Language changed to <green>English<white>!"
+  decorationItems:
+    - :
```

YAMLlint (1.37.1)

[warning] cannot infer indentation: unexpected token

(indentation)